initiatives/info.yml

```diff
@@ -0,0 +1,57 @@
+---
+resource: budgets
+query:
+  model: true
+  sql: |
+    select 
+        decidim_initiatives.id,
+        decidim_initiatives.id::text as id_as_text,
+        title->>'fr' as title,
+        regexp_replace(decidim_initiatives.description->>'fr', E'(<[^>]+>)|(&[a-z]+;)', '', 'gi') as description,
+        decidim_author_id,
+        published_at,
+        (case state
+          when 0 then 'Créée'
+          when 1 then 'Validée'
+          when 2 then 'Supprimée'
+          when 3 then 'Publiée'
+          when 4 then 'Rejetée'
+          when 5 then 'Acceptée'
+        end) as parsed_state,
+        state,
+        signature_type,
+        signature_start_date,
+        signature_end_date,
+        regexp_replace(answer->>'fr', E'(<[^>]+>)|(&[a-z]+;)', '', 'gi') as answer,
+        answered_at,
+        answer_url,
+        decidim_initiatives.created_at,
+        decidim_initiatives.updated_at,
+        decidim_user_group_id,
+        hashtag,
+        scoped_type_id,
+        decidim_author_type,
+        parsed_online_votes,
+        parsed_offline_votes,
+        parsed_online_votes + parsed_offline_votes as sum_votes,
+        comments_count,
+        follows_count,
+        concat('https://', (select host from {{#organizations}} org), '/initiatives/i-',decidim_initiatives.id) as url,
+        supports_required,
+        decidim_areas.id as area_id,
+        (case when decidim_areas.name ? 'fr' then decidim_areas.name->>'fr' else 'Pas de catégorie' end) as area_name,
+        (case when decidim_area_types.name ? 'fr' then decidim_area_types.name->>'fr' else 'Pas de type de catégorie' end) as area_type_name
```

Comment:
switch to `$LANGUAGE_CODE`

Reply:
fixed in [484818f](https://github.com/OpenSourcePolitics/decidim_cards/pull/65/commits/484818f4205f309fd8a3d8ff5282730828d94bb1)